src/RA_Integration.vcxproj

```diff
@@ -462,7 +462,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='AnalysisUnicode|Win32'">
     <ClCompile>
-      <PreprocessorDefinitions>_DEBUG;RA_EXPORTS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_DEBUG;RA_EXPORTS;_PREFAST_;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Visual Studio wasn't automatically defining `_PREFAST_` even though the Microsoft Recommended Ruleset is used..